### PR TITLE
IJPS-1 add Judicial-payment-service project

### PR DESCRIPTION
**Pull request by RebeccaBaker**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-1

### Change description ###
Add Judicial payment service project

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/jps/jps-judicial-payment-service/demo-image-policy.yaml
Comment (reespozzi) [inline apps/jps/jps-judicial-payment-service/demo-image-policy.yaml:1]: Is this needed so far if just referencing prod image?

File: apps/jps/perftest/base/kustomization.yaml
Comment (reespozzi) [inline apps/jps/perftest/base/kustomization.yaml:1]: Is this needed if no replicas?

Comment (RebeccaBaker) [reply apps/jps/perftest/base/kustomization.yaml:1]: removed

File: apps/jps/jps-judicial-payment-service/demo-image-policy.yaml
Comment (RebeccaBaker) [reply apps/jps/jps-judicial-payment-service/demo-image-policy.yaml:1]: removed

File: apps/jps/jps-judicial-payment-service/ithc.yaml
[the remark this answers is not in the document]
Comment (RebeccaBaker) [reply apps/jps/jps-judicial-payment-service/ithc.yaml:1]: removed

File: apps/jps/jps-judicial-payment-service/perftest.yaml
[the remark this answers is not in the document]
Comment (RebeccaBaker) [reply apps/jps/jps-judicial-payment-service/perftest.yaml:1]: removed

**Review by danlysiak — APPROVED** [review]

**Pull request merged**